languages/js/src/Query.ts

```diff
@@ -179,7 +190,7 @@ export class Query {
       )) as NullishOrHasConstructor;
       const rel = this.#host.getType(receiver?.constructor)?.fields?.get(attr);
       if (rel instanceof Relation) {
-        value = await this.handleRelation(receiver, rel);
+        value = await this.handleRelation(receiver as obj, rel);
```

Comment:
Won't `receiver` be a `Constructable` since `this.#host.getType(receiver?.constructor)?.fields?.get(attr)` evaluated to a `Relation` instance?

Reply:
We actually require it to be indexable with strings (ie. `obj`) but we test for it by seeing if it has a `constructor: Class`. Js being js means pretty much the latter => the former